react/features/base/flags/constants.ts

```diff
@@ -160,6 +160,12 @@ export const OVERFLOW_MENU_ENABLED = 'overflow-menu.enabled';
  */
 export const PIP_ENABLED = 'pip.enabled';
 
+/**
+ * Flag indicating if Picture-in-Picture button should be shown while translation.
```

Comment:
This description sounds very confusing. There is nothing related to translations here. What you want is to enable it while screen sharing.